.docker/Dockerfile

```diff
@@ -28,12 +26,15 @@ ENV NODE_VERSION       $NODE_VERSION
 RUN php -r "copy('https://getcomposer.org/installer', '/tmp/composer-setup.php');" && php /tmp/composer-setup.php --no-ansi --install-dir=/usr/local/bin --filename=composer && rm -rf /tmp/composer-setup.php
 
 RUN if [ "$INSTALL_XDEBUG" = "true" ]; then \
-        pecl install xdebug-3.1.3 \
-        && docker-php-ext-enable xdebug \
-        && echo "xdebug.mode=debug" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini \
-        && echo "xdebug.start_with_request=yes" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini \
-        && echo "xdebug.client_host=host.docker.internal" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini; \
-    fi
-
-COPY .docker/docker_run_git.sh /tmp/
+    pecl install xdebug-3.1.3 \
```

Comment:
Soon we should be able to replace this with `pie`, which is the official extension manager (should be voted tomorrow by the PHP foundation)